cmake/modules/CMakeFunctions.cmake

```diff
@@ -161,6 +162,8 @@ function(apply_patches repo_dir patches_dirs base_revision target_branch ret)
     endif()
     if (NOT (ret_not_git_repo OR ret_check_out OR ret_apply_patch))
         set(${ret} True PARENT_SCOPE)
+    else()
+        message(FATAL_ERROR "apply_patches failed!")
```

Comment:
I saw "[OPENCL-CLANG]" is used at beginning of messages above. Is it better: "[OPENCL-CLANG] Failed to apply patch!"?

Reply:
Done, thanks for your comments.